modules/mri_violations/help/mri_violations.md

```diff
@@ -0,0 +1,15 @@
+# MRI Protocol Violations
+
+The MRI Violations module displays scans that have been flagged for significant variance from the study protocol.
+Use the Selection Filters to specify or search by Patient Name, MincFile, Scan Type, Project, Site, then click Show Data
+to apply the selection filters in the data table below. 
+
+The data results table contains two tabs: "Not Resolved" (displayed by default) and "Resolved". The "Resolution Status"
+colun indicates and allows updates to the scan status: Reran, Emailed Site/Pending, Inserted, etc. When a scan is updated
```

Comment:
There's a typo in "column".. and I find "indicates and allows updates to the scan status" is pretty awkward phraseology, but I'm not sure how to fix it.

Reply:
Unawkwarding Suggestion: @llevitis 
The "Resolution Status" column is used to display and update the scan status: ..."